bridges/otellogr/logsink_test.go

```diff
@@ -30,44 +30,37 @@ func TestNewConfig(t *testing.T) {
 	customLoggerProvider := mockLoggerProvider{}
 
 	for _, tt := range []struct {
-		name    string
-		options []Option
-
-		wantConfig config
+		name     string
+		options  []Option
+		wantFunc func(config)
```

Comment:
This change is making the test rather harder to grasp. Let's not make it until we do need it because of context.

Reply:
Because when writing test cases, it is necessary to add some default attributes at any time, otherwise it may lead to some exceptions.

However, the focus of unit testing is not on these default attributes. Therefore, when adjusting the context (ctx) at that time, this part of the method was rewritten.

Reply:
Right, but we're not adjusting the context anymore. So there's no reason for this to be in the diff.